README.md

```diff
@@ -97,6 +97,7 @@ These books are for software developers with some experience in other languages,
 and willing to become familiar with C++
 
 * [A Tour of C++ 3rd Edition](https://www.amazon.com/dp/0136816487) (Bjarne Stroustrup, 2022)
+* [Beginning C++23](https://www.amazon.com/dp/1484293428) (Ivor Horton, Peter Van Weert, 2023)
```

Comment:
Beginning C++23 is a great book I've been recently reading which starts C++ from scratch while considering the most recent features of C++.
The previous edition was named Beginning C++20 so there is no need to include it in the list.
The book also doesn't belong to C++23 feature list because it doesn't just explain C++23 features but teaches C++ from scratch. Therefore, this is a good resource for beginners to start with.